### PR TITLE
Fix nits in CRD versioning docs

### PR DESCRIPTION
**Pull request by nikhita**
Fixes https://github.com/kubernetes/website/pull/8834#issuecomment-398310528

/assign sttts mistyhacks

### CODE REVIEW

File: content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md [hunk 1]
```diff
@@ -149,30 +149,30 @@ the version.
...
+or updated objects are written at the new storage version. It is possible for an
 object to have been written at a version that is no longer served.
 
 When you read an object, you specify the version as part of the path. If you
 specify a version that is different from the object's persisted version,
-Kubernetes returns the object to you at the version you requested, but does not
-modify the persisted object. You can request an object at any version that is
-currently served.
+Kubernetes returns the object to you at the version you requested, but
+the persisted object is not changed on disk and is returned without any conversion.
```
Comment (sttts) [inline content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md]: but the persisted object is neither changed on disk, nor converted in any way (other than changing the `apiVersion` string) while serving the request.

Comment (nikhita) [reply content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md]: Done.

**Pull request merged**